cmd/alias.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+
+	"github.com/tfversion/tfversion/pkg/alias"
+)
+
+const (
+	aliasExample = "# Alias a Terraform version\n" +
+		"tfversion alias default 1.7.4"
+)
+
+var (
+	aliasCmd = &cobra.Command{
+		Use:     "alias",
+		Short:   "Alias a Terraform version",
+		Example: aliasExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 2 {
+				fmt.Println("error: provide an alias name and Terraform version")
+				fmt.Printf("See %s for help and examples\n", color.CyanString("`tfversion alias -h`"))
+				os.Exit(1)
+			}
+			alias.AliasVersion(args[0], args[1])
+		},
+	}
+)
+
+func init() {
+	rootCmd.AddCommand(aliasCmd)
```

Comment:
I think that listing the workspaces is essential, can we add a flag `--list` or consider to add a subcommand to `tfversion list`, for instance: `tfversion list workspaces` - what are your thoughts on this?

Reply:
We should ensure that a normal `tfversion list --installed` only lists versions, and `tfversion list --aliases` could return a list of aliases and their target versions.

Reply:
Yeah that works for me 👍🏼

Reply:
I have solved this by creating a separate directory for actual version (`~/.tfversion/versions`) and aliases (`~/.tfversion/aliases`). This makes it easier to separate the two types. However current installations would need a migration (or we publish this as a breaking change).